opencog/cython/opencog/atom.pyx

```diff
@@ -39,6 +39,15 @@ cdef class Atom(Value):
                 else:
                     self._name = ""
             return self._name
+    property type:
```

Comment:
In C++ code the `type` is attribute of `Value` class, it is less surprising if this property is added into Python `Value` class as well.

Reply:
Hi, please see my second comment to the issue